.github/workflows/rfs_pr_e2e_test.yml

```
@@ -0,0 +1,40 @@
+name: Jenkins E2E
+
+on:
+  push:
+    branches-ignore:
+      - 'backport/**'
+      - 'dependabot/**'
+  pull_request_target:
+    types: [opened, synchronize, reopened]
+
+env:
+  python-version: '3.11'
+
+permissions:
+  contents: read # to fetch code (actions/checkout)
+
+jobs:
+  rfs-e2e-aws-test:
```

This doesn't read very clean to me, RFS is a technology, but the scenario is a backfill, what do you think of the following?
```suggestion
  migration-assistant-backfill-on-aws:
```

---

I'd like to have this match the pipeline name for now but I'm open to changing this in the future if we decide we don't like it